src/main/java/eu/dissco/core/translator/terms/specimen/event/EventAssertions.java

```diff
@@ -69,26 +73,59 @@ private Assertion createEventAssertion(
   private List<Assertion> gatherEventAssertionsForDwc(
       JsonNode data) {
     var assertions = new ArrayList<Assertion>();
-    if (data.get(EXTENSIONS) != null
-        && data.get(EXTENSIONS).get("dwc:MeasurementOrFact") != null) {
-      var measurementOrFactExtension = data.get(EXTENSIONS).get("dwc:MeasurementOrFact");
-      for (var jsonNode : measurementOrFactExtension) {
-        var assertion = new Assertion()
-            .withType("ods:Assertion")
-            .withDwcMeasurementUnit(
-                super.searchJsonForTerm(jsonNode, List.of("dwc:measurementUnit")))
-            .withDwcMeasurementType(
-                super.searchJsonForTerm(jsonNode, List.of("dwc:measurementType")))
-            .withDwcMeasurementValue(
-                super.searchJsonForTerm(jsonNode, List.of("dwc:measurementValue")))
-            .withDwcMeasurementRemarks(
-                super.searchJsonForTerm(jsonNode, List.of("dwc:measurementRemarks")));
-        assertions.add(assertion);
+    if (data.get(EXTENSIONS) != null) {
+      if (data.get(EXTENSIONS).get("dwc:MeasurementOrFact") != null) {
+        var measurementOrFact = data.get(EXTENSIONS).get("dwc:MeasurementOrFact");
+        mapMeasurementOrFact(measurementOrFact, assertions);
+      }
+      if (data.get(EXTENSIONS).get("http://rs.iobis.org/obis/terms/ExtendedMeasurementOrFact")
+          != null) {
+        var extendedMeasurementOrFact = data.get(EXTENSIONS)
+            .get("http://rs.iobis.org/obis/terms/ExtendedMeasurementOrFact");
+        mapMeasurementOrFact(extendedMeasurementOrFact, assertions);
       }
     }
     return assertions;
   }
 
+  private void mapMeasurementOrFact(JsonNode measurementOrFactExtension,
+      List<Assertion> assertions) {
+    for (var jsonNode : measurementOrFactExtension) {
+      var assertion = new Assertion()
+          .withType("ods:Assertion")
+          .withDwcMeasurementID(super.searchJsonForTerm(jsonNode, List.of("dwc:measurementID")))
+          .withDwcParentMeasurementID(
+              super.searchJsonForTerm(jsonNode, List.of("dwc:parentMeasurementID")))
+          .withDwcMeasurementUnit(
+              super.searchJsonForTerm(jsonNode, List.of("dwc:measurementUnit")))
+          .withDwciriMeasurementUnit(
+              super.searchJsonForTerm(jsonNode,
+                  List.of("http://rs.iobis.org/obis/terms/measurementUnitID")))
+          .withDwcMeasurementType(
+              super.searchJsonForTerm(jsonNode, List.of("dwc:measurementType")))
+          .withDwciriMeasurementType(
+              super.searchJsonForTerm(jsonNode,
```

Comment:
wouldn't these search fields be more appropriate in the Terms themselves?

Reply:
It's not wrong but this appears different from all the other terms, unless I'm missing something